Polyclinic/Polyclinic.API.Host/Controllers/AnalyticsController.cs

```diff
@@ -0,0 +1,154 @@
+using Microsoft.AspNetCore.Mvc;
+using Polyclinic.Application.Interfaces;
+using Polyclinic.Contracts.Dto;
+
+namespace Polyclinic.API.Host.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+[Produces("application/json")]
+public class AnalyticsController(
+    IAnalyticsService analyticsService,
+    ILogger<AnalyticsController> logger) 
+    : ControllerBase
+{
+
+    [HttpGet("doctors/experience/{years:int}")]
+    [ProducesResponseType(typeof(List<DoctorDto>), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status500InternalServerError)]
```

Comment:
Вы работаете с ActionResult и тут нет необходимости указания типа. Можно упростить:
```suggestion
    [ProducesResponseType(StatusCodes.Status200OK)]
    [ProducesResponseType(StatusCodes.Status400BadRequest)]
    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```